contracts/src/card.cairo

```diff
@@ -0,0 +1,55 @@
+/// Represents a playing card. It only contains the token id of the NFT.
+#[derive(Component, Copy, Drop, Serde, SerdeLen)]
+struct Card {
```

Comment:
Hmm I think we can key this by the token id. We probably want it to contain player stats?

Reply:
idk if the player stats should be at this level or at the nft contract level